plugins/reporters/web-app-template/src/models/WebAppPackage.js

```diff
@@ -62,6 +62,8 @@ class WebAppPackage {
 
     #detectedLicensesProcessed = new Set();
 
+    #detectedLicensesProcessedIndexes = new Set();
```

Comment:
Commit message nit: Put `detectedLicensesProcessedIndexes` and `EvaluatedModel` in backticks.